source/query-with-natural-language/enable-natural-language-querying.txt

```diff
@@ -0,0 +1,90 @@
+.. _enable-natural-language-querying:
+
+================================
+Enable Natural Language Querying
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+You can enable Compass natural language querying by toggling the 
+:guilabel:`Use Generative AI` setting. Natural language queries can 
+be a helpful starting point and assist you in learning to write MongoDB 
+queries.
+
+About this Task
+---------------
+
+Alternatively, you can enable natural language querying by clicking the 
+:guilabel:`Log in to Atlas to enable` button within the 
+:guilabel:`Use natural language to generate queries and pipelines` modal. 
+This modal displays after updating Compass to a version that supports 
+natural language querying and clicking the :guilabel:`Generate` 
```

Comment:
This will say "Generate Query" or "Generate Aggregation" depending on the context. Is it worth spelling this out?